crates/astria-telemetry/src/lib.rs

```diff
@@ -219,6 +280,21 @@ impl Config {
             .try_init()
             .map_err(Error::init_subscriber)?;
 
+        if let Some(metrics_addr) = metrics_addr {
+            let addr: SocketAddr = metrics_addr.parse().map_err(Error::metrics_addr)?;
+            let mut metrics_builder = PrometheusBuilder::new()
+                .with_http_listener(addr)
+                .add_global_label("service", service_name);
```

Comment:
So if one were to leave `service_name` unset then there would be a global label `"service"` with value `""`? Does that make sense or would that be confusing?

Reply:
Updated to only set if service_name is not empty string.